PullRequestCreated/PullRequestCreated.ts

```diff
@@ -1,27 +1,20 @@
 import { OctokitAction } from '../lib/OctokitAction';
 import { PullRequest } from '../lib/OctokitTypes';
-import { JIRA_ISSUE_PATTERN } from '../lib/Constants';
+import { JIRA_ISSUE_PATTERN, JIRA_TASK_ISSUE } from '../lib/Constants';
 
 class PullRequestCreated extends OctokitAction {
   protected async execute(): Promise<void> {
     const pr = await this.getPullRequest(this.payload.pull_request.number);
-    if (pr == null) {
-      console.log('Pull request not found.');
-      return;
-    }
     if (this.shouldCreateIssue(pr)) {
       const projectKey = this.getInput('jira-project');
-      const issueType = await this.jira.findIssueType(projectKey, 'Task');
-      if (issueType != null) {
-        const issueKey = await this.jira.createIssue(projectKey, issueType, pr.title);
-        this.updatePullRequestTitle(this.payload.pull_request.number, `${issueKey} ${pr.title}`);
-        this.updatePullRequestDescription(this.payload.pull_request.number, `${issueKey}\n\n${pr.body || ''}`);
-      }
+      const issueKey = await this.jira.createIssue(projectKey, JIRA_TASK_ISSUE, pr.title);
+      await this.updatePullRequestTitle(this.payload.pull_request.number, `${issueKey} ${pr.title}`);
+      await this.updatePullRequestDescription(this.payload.pull_request.number, `${issueKey}\n\n${pr.body || ''}`);
     }
   }
 
   private shouldCreateIssue(pr: PullRequest): boolean {
-    return !JIRA_ISSUE_PATTERN.test(pr.title);
+    return !JIRA_ISSUE_PATTERN.test(pr?.title);
```

Comment:
If the `getPullRequest` returns null, the `test(null)` returns `false` and then `!false` will return `true`, resulting in an issue creation with empty summary. 

```suggestion
    return pr != null && !JIRA_ISSUE_PATTERN.test(pr.title);
```